packages/dashboard/src/components/ConversationDetailModal.tsx

```
@@ -0,0 +1,400 @@
+            {conversation.outcome}
+          </span>
+          {conversation.comment && (
+            <div className="mt-2">"{conversation.comment}"</div>
+          )}
+        </div>
+      </div>
+    </div>
+  </div>
+)
+
+const ConversationContext: React.FC<{ conversation: ConversationSummary; context: ConversationContext }> = ({ conversation, context }) => (
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ConversationContext'. Consider to delete it or rename it.

'ConversationContext' is defined here:


(lint/suspicious/noRedeclare)